SPECS/opentelemetry-cpp/opentelemetry-cpp.spec

```diff
@@ -45,6 +50,7 @@ mkdir build && cd build
 	-DWITH_NO_DEPRECATED_CODE=ON \
 	-DWITH_OTLP_GRPC=ON \
 	-DWITH_OTLP_HTTP=ON \
+	-DWITH_ABSEIL=ON \
```

Comment:
I assume this means "With the System Abseil" and not "with the embedded and vendored abseil?"

Reply:
Opentelemetry documentation says:

> gRPC internally uses a different version of Abseil than OpenTelemetry C++ SDK.
> One option to optimize your code is to build the SDK with system-provided
> Abseil library. If you are using CMake, then `-DWITH_ABSEIL=ON` may be passed
> during the build of SDK to reuse the same Abseil library as gRPC.

Reply:
Without this opentelemetry wasn't picking up our abseil-cpp which it needed after the upgrade